staff/emma-cowdery/blog/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>The dog blog</title>
+</head>
+<body>
+    <header>
+        <h1>The dog blog</h1>
+    </header>
+    <nav>
+            <ul>   
+                <li>About</li>
+                <li>Vests</li>
+                <li>Treats</li>
+                <li>Wellness</li>
+                <li>Gear</li>
+                <li>Blog</li>
+                <li>Deals</li>
+            </ul>
+    </nav>
+    <article>
+        <section>
+            <ul>
```

Comment:
Maybe you could think of deleting the ul tag and changing each li tag for an article tag.